### ISSUE TITLE
Kunoros, Hound of Athreos does not prevent reanimation spells that affect multiple creatures at the same time depending on graveyard order

### ISSUE THREAD

**Issue by tritonicon** [PennyDreadfulMTG/modo-bugs#878]
**What is the expected behaviour of the card?**
 - Kunoros, Hound of Athreos, while in a graveyard, should not effect reanimation spells or abilities. Kunoros's ability should only be in effect while it is on the battlefield.

**What is the actual behaviour of the card?**
 - Kunoros, Hound of Athreos effects what can be mass reanimated by spells, preventing cards underneath it in the pile from being reanimated or similar hard to follow behavior. 
**Screenshot**
Reported from trusted friend, though has been reported by others before. See: 

**Please provide a list of affected cards**
Affects: [Kunoros, Hound of Athreos]
See Also: [Command the Dreadhorde], [Eerie Ultimatum], [Rise of the Dark Realms], and [Magister of Worth] or presumable any other mass return card to battlefield spell involving creatures. Single target/choice spells and abilities not tested yet. 

**Card Images**
 <img src="https://pennydreadfulmagic.com/image/Kunoros%2C%20Hound%20of%20Athreos/" height="300px"><img src="https://pennydreadfulmagic.com/image/Command%20the%20Dreadhorde|Eerie%20Ultimatum|Rise%20of%20the%20Dark%20Realms|Magister%20of%20Worth/" height="250px">

Support Thread: https://feedback.wizards.com/forums/918667-mtg-arena-bugs-product-suggestions/suggestions/39654034-kunoros-hound-of-athreos-command-the-dreadhorde

**Comment by MultiDragon** [comment]
Affects [Magister of Worth] as well. Also add [Kunoros, Hound of Athreos] to the list of affected cards otherwise the bot wouldn't find the bug

**Comment by tritonicon** [comment]
Added Magister of Worth of worth to list of Affected cards and changed wording slightly, including that single target or choice spells and abilities have not been tested yet.

**Comment by jmeka** [comment]
This seems to be fixed. 
![Screenshot 2021-02-10 181032](https://user-images.githubusercontent.com/28836887/107594105-26eedf00-6bdf-11eb-9895-dd048203dcbf.png)

**Comment by Kalioc** [comment]
![image](https://user-images.githubusercontent.com/9436442/107594698-8b5e6e00-6be0-11eb-8ccb-653dd88fb5a3.png)
Still bugged, the description is just not very good. Attempting to mass reanimate cards, with Kunoros among those cards, seems to cause cards either below or above it to not enter the battlefield. In this screenshot, all other cards were under Kuronos and they didn't enter the battlefield.

**Comment by Kalioc** [comment]
![image](https://user-images.githubusercontent.com/9436442/107595151-d6c54c00-6be1-11eb-8a7e-4921e44b03fb.png)
In this screenshot, Undead Augur was above Kuronos and was the only one to not enter the battlefield.

**Comment by Kalioc** [comment]
I was looking for this and didn't realize it was closed. Was confirmed broken on the day it was closed and probably is still broken, just had a match where cards still didn't enter from the graveyard at the same time.

**Comment by datrobino** [comment]
duplicate
